requirements.txt

```diff
@@ -1,15 +1,12 @@
 numpy
 accelerate>=0.20.3
-packaging
-transformers>=4.34.1
+transformers
 torch
-aim==3.18.1
+aim==3.17.5
```

Comment:
why is this lowered?

Reply:
This changed in [PR 42](https://github.com/foundation-model-stack/fms-hf-tuning/pull/42) after I had started my PR.  I'm not sure why it didn't come up as a merge conflict.  Will fix it and re-rest everything, make sure it all still works.

Reply:
ok - reverted back, thanks!